packages/config-utils/proxy.js

```diff
@@ -25,15 +25,16 @@ module.exports = ({
     localChrome,
     appUrl = [],
     publicPath,
-    proxyVerbose
+    proxyVerbose,
+    useCloud = false
```

Comment:
In order to make adopting the webpack proxy easier and smooth out the transition can we please set this to `true`? Currently it's **required** to pass `useCloud = true` since console.redhat.com is not working.

Reply:
Well the transition to console.redhat.com should happen in a few days, we'd have to open another PR to flip this value, apps would then have to update their config version once again. This is safer way, if you are going to use newer version add `useCloud: true` to config, once the console is deployed you can just remove that from the config. This approach means less steps for applications (they'd have to update the package in order for them to work without any problem).

Reply:
It currently means more steps for applications I have to deal with, but I'll trust you.